.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```

Comment:
instead of delete, did you want them to select the appropriate option(s)

Reply:
Fixed, thanks